File: frontend/src/types/discharge.ts
import type { DischargeBreakdown } from "@/types/common"

/** Mirrors backend/app/schemas/discharge.py exactly. */
export interface DischargeSummary {
  discharged: number
  referred: number
  lama: number
  death: number
  still_admitted: number
}

export interface DischargeResponse {
  summary: DischargeSummary
  inborn: DischargeBreakdown
  outborn: DischargeBreakdown
}

File: backend/app/schemas/discharge.py
from pydantic import BaseModel

from app.schemas.common import DischargeBreakdown


class DischargeSummary(BaseModel):
    discharged: int
    referred: int
    lama: int
    death: int
    still_admitted: int


class DischargeResponse(BaseModel):
    summary: DischargeSummary
    inborn: DischargeBreakdown
    outborn: DischargeBreakdown
